### DISCUSSION TITLE
Where did the data in iraf.noao.edu disappear?

### DISCUSSION THREAD

**Discussion by seyithokelek** [iraf-community/iraf#204] [Q&A]
Where did the data in iraf.noao.edu disappear?

http://iraf.noao.edu/specatlas/fear/fear.html

How do I access the fear atlas on this website?
Does anyone have information?

**Comment by olebole** [comment]
http://iraf.noao.edu was officially stopped in 2019. Unofficially it however partially worked until last month or so, when NOAO changed its name and its website URL to noirlab.edu. Since then, all noao.edu hosts are unreachable.

So, I guess that the fear atlas is finally gone.

With some luck, you may someone from noirlab may have a solution for you, but NOAO officially ended all its IRAF support two years ago.

**Comment by kortizceballos** [comment]
I had a similar issue as I also needed to access specatlas, and found that it's still somewhat available in Wayback Machine snapshots. For the fear atlas, the most recent one is from 2021 and can be found at [http://web.archive.org/web/20210311235423/http://iraf.noao.edu/specatlas/fear/fear.html](http://web.archive.org/web/20210311235423/http://iraf.noao.edu/specatlas/fear/fear.html). It seems that both the atlas fits file and the complete line list are downloadable from the archive. Hope this helps.
